pkg/liqo-controller-manager/namespaceMap-controller/ensure_remote_namespaces.go

```diff
@@ -228,28 +228,40 @@ func (r *NamespaceMapReconciler) namespaceMapDeletionProcess(ctx context.Context
 
 // checkRemoteNamespaceRoleBindings checks that the right roleBindings are inside the remote namespace to understand if the
 // virtual kubelet will have the right privileges on that namespace.
-func checkRemoteNamespaceRoleBindings(ctx context.Context, cl kubernetes.Interface,
```

Comment:
I am personally missing why this change is required.

Reply:
As for the first few rows: because the role binding labels refers to the tenant, which depends on the cluster name set by (I believe) the remote cluster, which is unknown (it is not necessarily the same as the cluster name of the local installation).

For the second half, I figured listing the role bindings explicitly was much clearer than expecting 3 bindings.

Reply:
Can this conversation be marked as resolved?